src/bin/main.rs

```diff
@@ -157,39 +89,54 @@ fn exec_command(args: Vec<String>, setcap: bool) {
 }
 
 fn main() {
-    match std::env::current_exe() {
-        Ok(exe) => {
-            if exe == Path::new(cntr::SETCAP_EXE) {
-                exec_command(env::args().collect::<Vec<String>>(), true)
-            }
-        }
-        Err(e) => {
-            eprintln!("failed to resolve executable: {}", e);
-            process::exit(1);
-        }
-    }
```

Comment:
This part need to be kept. If the current executable is an capability executable wrapper it should execute the binary right away.

Reply:
fixed in d05b1be. Should i resolve the merge conflict in Cargo.toml now to allow merge?